frontend/gotgt/frontend.go

```diff
@@ -136,15 +157,24 @@ func (t *goTgt) startScsiTarget(cfg *config.Config) error {
 	var err error
 	id := uuid.NewV4()
 	uid := binary.BigEndian.Uint64(id[:8])
-	scsi.InitSCSILUMapEx(t.tgtName, t.Volume, uid, 0, uint64(t.Size), uint64(t.SectorSize), t.rw)
+	err = scsi.InitSCSILUMapEx(&config.BackendStorage{
+		DeviceID:         uid,
+		Path:             "RemBs:" + t.tgtName,
+		Online:           true,
+		BlockShift:       9,
+		ThinProvisioning: true,
+	},
+		t.tgtName, uint64(0), t.rw)
+	if err != nil {
+		return err
+	}
 	scsiTarget := scsi.NewSCSITargetService()
 	t.targetDriver, err = scsi.NewTargetDriver("iscsi", scsiTarget)
 	if err != nil {
 		logrus.Errorf("iscsi target driver error")
 		return err
 	}
 	t.targetDriver.NewTarget(t.tgtName, cfg)
-	t.targetDriver.SetClusterIP(t.clusterIP)
```

Comment:
Is this ClusterIP set in some other way? Without this we can't use kubernetes service IP to access the target.

Reply:
@kmova target runs in a Pod, so it runs on "localhost" which is then redirected by service IP

Reply:
So we don't need to set clusterIP

Reply:
anyway i will test this to verify